integration/tests/cook/test_cli.py

```diff
@@ -771,7 +771,7 @@ def entry(name):
         self.logger.debug(entries)
         foo = entry('foo')
         self.assertEqual('drwxr-xr-x', foo['mode'])
-        self.assertEqual(2, foo['nlink'])
+        self.assertGreaterEqual(2, foo['nlink'])
```

Comment:
This should be `assertLessEqual` if `foo['nlink']` is sometimes greater than 2.

Reply:
Fixed, thanks